docs/gallery/plot_external_resources.py

```diff
@@ -306,6 +307,52 @@ def __init__(self, **kwargs):
     entity_uri='https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=10090'
 )
 
+###############################################################################
+# Using add_ref_termset
+# ------------------------------------------------------
+# :py:class:`~hdmf.common.resources.HERD` has multiple ways for users to add
+# external references. The :py:func:`~hdmf.common.resources.HERD.add_ref_termset`
+# method allows users to not only validate terms, i.e., keys, but also adds the
+# ability to iteratively add references for entire datasets, lists, arrays, etc.
+
+# The :py:func:`~hdmf.common.resources.HERD.add_ref_container` method is directly
+# used for populating :py:class:`~hdmf.common.resources.HERD` when writing a file.
+# :py:func:`~hdmf.common.resources.HERD.add_ref_termset` can be used for new files;
+# however, it is also the best practice when adding references for existing files.
+
+# :py:func:`~hdmf.common.resources.HERD.add_ref_termset` has many optional fields,
+# giving the user a range of control when adding references. Let's see an example.
+er = HERD()
```

Comment:
```suggestion
er = HERD()
```
Can we name the variable `er` to `herd` throughout these examples? I don't think `er` would mean anything to a new user, now that we have renamed everything to HERD.